src/main/kotlin/com/example/minibank/controllers/AuthController.kt

```diff
@@ -81,6 +119,13 @@
     @PostMapping("/login")
     @Operation(summary = "Авторизация пользователя")
     fun login(@RequestBody body: UserDTO, response: HttpServletResponse): ResponseEntity<Any> {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty